demos/common/cpp_gapi/utils_gapi/src/kernel_package.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2021-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "utils_gapi/kernel_package.hpp"
+
+namespace util {
+cv::gapi::GKernelPackage getKernelPackage(const std::string& type) {
```

Comment:
This is not needed if we're going to use `cpu` backend only

Reply:
are we going?

by the way - it's `utility` common place, which is usable by other gapi demo